packages/contracts-bedrock/tasks/generate-deploy-config.ts

```diff
@@ -0,0 +1,28 @@
+import fs from 'fs'
+import { task } from 'hardhat/config'
+import { HardhatRuntimeEnvironment } from 'hardhat/types'
+
+task(
+  'generate-deploy-config',
+  'generates a json config file for the current network'
+).setAction(async ({}, hre: HardhatRuntimeEnvironment) => {
+  try {
+    const base = `${hre.config.paths.deployConfig}/${hre.network.name}`
+    if (fs.existsSync(`${base}.ts`)) {
+      // eslint-disable-next-line @typescript-eslint/no-var-requires
+      const config = require(`${base}.ts`).default
+      console.log('config:', config)
```

Comment:
perhaps no logging of the full config here? especially since its shared with a string in the same log

Reply:
Whoops, debug code left in.